### ISSUE TITLE
Server stop working in specific world if loaded with phosphor 

### ISSUE THREAD

**Issue by NikitaCartes** [Gegy/tic-tacs#22]
Step to reproduce:
First world:
1) Downloads the world
2) Open it.
You will get this and loading stops.
```
[10:59:36] [lighting-worker/INFO]: [STDERR]: java.lang.NullPointerException
[10:59:36] [lighting-worker/INFO]: [STDERR]: 	at net.minecraft.class_3556.method_15502(class_3556.java:537)
[10:59:36] [lighting-worker/INFO]: [STDERR]: 	at net.minecraft.class_3560.method_15525(class_3560.java:670)
[10:59:36] [lighting-worker/INFO]: [STDERR]: 	at net.minecraft.class_3558.method_15485(class_3558.java:156)
[10:59:36] [lighting-worker/INFO]: [STDERR]: 	at net.minecraft.class_3554.method_15492(class_3554.java:215)
[10:59:36] [lighting-worker/INFO]: [STDERR]: 	at net.minecraft.class_3558.method_15516(class_3558.java:193)
[10:59:36] [lighting-worker/INFO]: [STDERR]: 	at net.minecraft.class_3568.method_15563(class_3568.java:59)
[10:59:36] [lighting-worker/INFO]: [STDERR]: 	at net.minecraft.class_3227.method_15563(class_3227.java:1050)
[10:59:36] [lighting-worker/INFO]: [STDERR]: 	at net.gegy1000.tictacs.async.worker.LightingExecutor.runTasks(LightingExecutor.java:65)
[10:59:36] [lighting-worker/INFO]: [STDERR]: 	at net.gegy1000.tictacs.async.worker.LightingExecutor.run(LightingExecutor.java:51)
[10:59:36] [lighting-worker/INFO]: [STDERR]: 	at java.lang.Thread.run(Unknown Source)
```
Second world:
1) Downloads the world
2) Open it.
3) After world ends loading, you will have same error. (If chunks radius >= 16).
4) If you try loads new chunks after that, world would stop working. (F.E. Nether Portal)

tic-tacs-0.1.2.jar
phosphor-fabric-mc1.16.4-0.6.0+build.64.jar (from https://github.com/jellysquid3/phosphor-fabric/pull/22)
fabric-api-0.25.1+build.416-1.16.jar

[World_1.zip](https://github.com/Gegy/tic-tacs/files/5535498/World_1.zip)
[World_2.zip](https://github.com/Gegy/tic-tacs/files/5535499/World_2.zip)

UPD: tic-tacs without phosphor or phosphor without tic-tacs works fine.
UPD-2: Crash-report from server, but slightly different setup
[crash-2020-11-13_10.27.25-server.txt](https://github.com/Gegy/tic-tacs/files/5535708/crash-2020-11-13_10.27.25-server.txt)

**Comment by Gegy** [comment]
The Phosphor dev branch is currently incompatible with Tic-TACS. Not entirely sure if it'll be solvable unfortunately, but hopefully can get it working eventually.

**Comment by NikitaCartes** [comment]
That's understandable, thanks. And without Performance without Phosphor is pretty good too, so it isn't such a big deal.
And big thanks for such a cool mod, 96 render distance looks amazing.


In additional to original issues:
After that I disabled Phosphor and all works fine until one player re-entered. After 60 seconds (without any exceptions or error in log) watchdog killed the server. After startup this player tried to login and it happened again. After that I disable and tic-tacs and server didn't die after player's logged.

I tried to replicate this in singleplayer (downloads region files around player's coords), but nothings happened and all works fine.

I can attach crash-report, but I not sure what this mod cause the problem.
